resources/templates/md/add-basic-search.md

```diff
@@ -0,0 +1,118 @@
+{:title "Basic Search with Lunr"
+ :layout :page
+ :toc true
+ :page-index 13
+ :section "Additional Resources"}
+
+To enable a search option for your site, you can use [`Lunr`](https://lunrjs.com/), which will create a searchable index without the need of any server-side or external search service. The inspiration to integrate Lunr with Cryogen was the following site [`Lunr-Index-and-Search-for-Static-Sites`](https://github.com/BLE-LTER/Lunr-Index-and-Search-for-Static-Sites), where the source Javascript files can be found, which are used with slightly modifications here.
+
+A life demo of the result can be found at: [https://blog.stderr.at/](https://blog.stderr.at/)
```

Comment:
"live"

Reply:
now thats embarrassing ... I hope my english teacher does not read this